src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SuperFinalizeCheck.java

```diff
@@ -23,20 +23,22 @@
 
 /**
  * <p>
- * Checks that an overriding finalize() method invokes super.finalize().
+ * Checks that an overriding {@code finalize()} method invokes {@code super.finalize()}.
  * Does not check native methods, as they have no possible java defined implementation.
  * </p>
  * <p>
- * Reference:<a
- * href="https://javarevisited.blogspot.com/2012/03/finalize-method-in-java-tutorial.html">
- * 10 points on finalize method in Java</a>.
+ * Reference: <a
+ * href="https://www.oracle.com/technetwork/java/javamail/finalization-137655.html">
+ * Use Finalization Only When You Must</a>.
```

Comment:
It seems these URLs use to be the same until they were changed to different locations and both weren't updated.
https://github.com/checkstyle/checkstyle/commit/a0a63e313fc878267a5d41c89b144e0d50e7bd0b#diff-8123de5694bfa2580e6ae4cb1934cf0bR29
https://github.com/checkstyle/checkstyle/commit/e8756de8ec385e6cedaa9e992ccdf61ae0b1e4fc#diff-7b7a98a340dc4db79fa5029ae03bd945R1488

Reply:
Both links are still alive. I think we should keep all of them.